cmd/serverless-init/initcontainer/initcontainer.go

```diff
@@ -51,11 +52,15 @@ func execute(cloudService cloudservice.CloudService, config *serverlessLog.Confi
 
 	cmd := exec.Command(commandName, commandArgs...)
 	cmd.Stdout = &serverlessLog.CustomWriter{
-		LogConfig: config,
+		LogConfig:  config,
+		LineBuffer: bytes.Buffer{},
+		IsDotnet:   commandName == "dotnet",
```

Comment:
nit: maybe we can change this to something like `shouldBuffer` instead of `isDotnet`?